boards/arduino-mkr1000/Makefile.include

```diff
@@ -0,0 +1,35 @@
+# define the cpu used by Arduino/Genuino MKR1000 board
+export CPU = samd21
+
+ifeq ($(PROGRAMMER),jlink)
+    # in case J-Link is attached to SWD pins, use a plain CPU memory model
+    export CPU_MODEL = samd21g18a
+
+    # setup JLink for flashing
+    export JLINK_DEVICE := atsamw25
+    include $(RIOTMAKE)/tools/jlink.inc.mk
+
+    # set default serial port depending on operating system
+    PORT_LINUX ?= /dev/ttyUSB0
```

Comment:
I copied [Makefile from stm32f4 discovery](https://github.com/RIOT-OS/RIOT/blob/master/boards/stm32f4discovery/Makefile.include#L5). Just like the case in stm32f4 discovery, `/dev/ttyUSB0` is for UART serial converter. But you are right - this is independent to the JLink. I added this when I didn't know how `bossac` works.